core-web/libs/edit-content/src/lib/fields/dot-edit-content-relationship-field/components/dot-select-existing-content/components/search/search.component.html

```diff
@@ -1,5 +1,12 @@
 <form [formGroup]="form" #formElement>
-    <p-inputGroup>
+    <p-inputGroup styleClass="flex align-items-center">
+        @for (filter of $activeFilters(); track filter.value; let first = $first) {
```

Comment:
The let first variable is not necessary with the new Angular `@for`; using `$first `is enough.